brainglobe_stitch/stitching_widget.py

```diff
@@ -370,12 +429,20 @@ def _on_stitch_button_clicked(self) -> None:
             display_info(self, "Warning", error_message)
             return
 
-        self.image_mosaic.stitch(
+        worker = create_worker(
+            self.image_mosaic.stitch,
             self.imagej_path,
             resolution_level=2,
             selected_channel=self.fuse_channel_dropdown.currentText(),
         )
 
+        self.fuse_button.setEnabled(False)
+        self.stitch_button.setEnabled(False)
+        self.adjust_intensity_button.setEnabled(False)
+        worker.finished.connect(self._on_stitch_finished)
+        worker.start()
```

Comment:
Sincerely, I don't understand why it is not re-stitching, there are no filters controlling for saved values 🤔